src/vaadin-context-menu-overlay.html

```diff
@@ -69,6 +69,12 @@
             parentOverlay: {
               type: Object,
               readOnly: true
+            },
+
+            theme: {
```

Comment:
This shouldn't be added. It's already defined in theme property mixin

Reply:
Removed.

Reply:
No observer was added, what invokes `_themeChanged` now?